src/main/resources/config/application.yml

```diff
@@ -162,7 +162,7 @@ spring:
       client:
         provider:
           oidc:
-            issuer-uri: http://keycloak:9080/auth/realms/jhipster
+            issuer-uri: http://localhost:9080/auth/realms/jhipster
```

Comment:
Das hier macht evtl. unsere OIDC-config kaputt. @dgwetzel?

Reply:
ja genau, das würde Probleme machen.

Als @Well5a die MediaService erzeugt hat, hatte ich noch nicht die aktuelle Version von jdl gepusht, daher vermutlich hier die "vielen" Änderungen.

Ich würde fast sagen, die Änderungen von dem pr können wir verwerfen oder ?

Reply:
Ich bin mir nicht sicher, ob das dann die Verbindung zum Mediaservice kaputt macht, kann nicht einschätzen was davon wichtig ist.

Reply:
an dieser Stelle ehe weniger, die Services sind ja unabhängig voneinander

Reply:
Dann bin ich dafür, den Pull Request und die Änderungen zu verwerfen

Reply:
Irgendwas in dem Commit bewirkt außerdem den gleichen "Illegal Reflective Access"-Error, den das MediaService-Repo auch hat. Daher bitte das Branch noch nicht entfernen, weil uns das hier vielleicht Anhaltspunkte liefert, wieso MediaService-Builds noch nicht tun.